user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Manually_Connecting_DMA_to_Elasticsearch_Cluster.md

```diff
@@ -0,0 +1,36 @@
+---
+uid: Manually_Connecting_DMA_to_Elasticsearch_Cluster
+---
+# Manually connecting a DMA to an Elasticsearch cluster
+
+You can connect a DataMiner Agent to an existing Elasticsearch cluster by manually adding a *DataBase* tag to the *DB.xml* file, as detailed below.
+
+> [!IMPORTANT]
+> Make sure the DataMiner Agent has an available connection to each node of the Elasticsearch cluster. To verify this, enter `http://elasticnodeip:9200/` in your browser's address bar to access the general server information.
+>
+> ![Admin consent](~/user-guide/images/ElasticSearch_Connectivity_Check.png)
+
+1. Stop the DataMiner Agent. See [Starting or stopping DataMiner Agents in your DataMiner System](xref:Starting_or_stopping_DataMiner_Agents_in_your_DataMiner_System).
+
+1. In the *C:\\Skyline DataMiner\\Database* directory, find the *DB.xml* file and copy this to your local machine as a backup.
+
+1. Open the original *DB.xml* file.
+
+1. Locate the active *DataBase* tag for Elasticsearch and specify the hosts, username, and password.
+
+   > [!TIP]
+   > See [Specifying custom credentials for Elasticsearch](xref:DB_xml#specifying-custom-credentials-for-elasticsearch).
+
+   Example:
+
+   ```xml
+   <DataBase active="true" search="true" type="Elasticsearch">
+      <DBServer>Node IP1,Node IP1,..,Node IPx</DBServer>
+      <UID>elastic</UID>
+      <PWD>{220CCF11-0785-4712-BBAB-E7DD72CFB0D6}</PWD>
+   </DataBase>
+   ```
+
```

Comment:
Might want to leave the password tag empty here. By default it's empty and some users would copy the GUID from other DMAs, which doesn't work, since it's not a password, merely a reference to an encrypted one. From Core Eco's side we should probably document the password setting and encryption process better, but this would be separate from this article still.